src/containers/InlinePlayer/InlinePlayer.tsx

```diff
@@ -16,22 +24,59 @@ type Props = {
   liveStartDateTime?: string | null;
   liveEndDateTime?: string | null;
   liveFromBeginning?: boolean;
+  startWatchingButton: React.ReactNode;
+  isLogged: boolean;
+  isLocked: boolean;
 };
 
-const InlinePlayer: React.FC<Props> = ({ open, item, onPlay, onPause, onComplete, feedId, liveStartDateTime, liveEndDateTime, liveFromBeginning }: Props) => {
+const InlinePlayer: React.FC<Props> = ({
+  open,
```

Comment:
The `open` prop is a bit weird for the inline player. I also think it is redundant actually